vm/default_runtime/src/lib.rs

```diff
@@ -546,3 +513,54 @@ fn transfer<ST: StateTree>(
 
     Ok(())
 }
+
+/// Returns public address of the specified actor address
+pub fn resolve_to_key_addr<'st, 'bs, ST, BS>(
+    st: &'st ST,
+    store: &'bs BS,
```

Comment:
Just a thought: Seems weird that there needs to be two references to the blockstore here, I guess way around this is to not have the state tree generic and use HamtStateTree specifically which isn't necessarily better and this is probably better for future proofing, but just figured I'd mention